src/app/wizards/dataModel/data-model-main/data-model-main.component.ts

```diff
@@ -34,11 +36,18 @@ export class DataModelMainComponent implements OnInit {
   isLinear = false;
   steps: Step[] = [];
   doneEvent = new EventEmitter<any>();
-  parentFolderId: any;
+  parentFolderId: string;
   parentFolder: any;
-  model: any = {
-    metadata: [],
-    classifiers: []
+  model : {
+    metadata: Array<any>;
+    classifiers: Array<any>;
+    label:string | undefined;
```

Comment:
👍 for adding additional types. It is also possible to say that `string | undefined` is the same as `label?: string` - they are equivalent.

I'm also trying to add a lot of these types into `mdm-resources` so that the UI does no have to keep redefining them.